### ISSUE TITLE
Questions about adaptive diffusion.

### ISSUE THREAD

**Issue by hot-dog** [Zhendong-Wang/Diffusion-GAN#36]
As the paper state, the metric rd that estimates how much the discriminator overfits to the data is defined as rd=E[sign(D(y, t) - 0.5)]. But when it comes to the implementation of the code, rd is defined as rd=E[sign(D(y, t))], without subtracting 0.5. My question is if i switch to a different GAN, such as LSGAN or Vanilla GAN, whose loss function is different from StyleGAN2, can I still use the rt defined above? Do I need to make appropriate modifications to rt?

**Comment by Zhendong-Wang** [comment]
Answered [here](https://github.com/Zhendong-Wang/Diffusion-GAN/issues/14#issuecomment-1406049513).

**Comment by hot-dog** [comment]
Thank you very much for your prompt reply, i understand.
